### ISSUE TITLE
fails to build with go 1.10 prerelease

### ISSUE THREAD

**Issue by decathorpe** [chmduquesne/rollinghash#9]
```
go test github.com/chmduquesne/rollinghash
# github.com/chmduquesne/rollinghash_test
(...)/src/github.com/chmduquesne/rollinghash/doc_test.go:36: Fatalf format %s has arg classic.Sum32() of wrong type uint32
FAIL	github.com/chmduquesne/rollinghash [build failed]
```

**Comment by chmduquesne** [comment]
Thanks for the bug report! I will have a look ASAP.

**Comment by chmduquesne** [comment]
Closed by abb8cba

**Comment by decathorpe** [comment]
Thanks for fixing it! :+1:
